helmfile.d/0410.grafana.yaml

```diff
@@ -34,12 +34,8 @@ releases:
         create: {{ env "GRAFANA_SERVICE_ACCOUNT_CREATE" | default "false" }}
         ### Optional: GRAFANA_SERVICE_ACCOUNT_NAME;
         name: '{{ env "GRAFANA_SERVICE_ACCOUNT_NAME" }}'
-      persistence:
```

Comment:
Why did we drop this?

Reply:
Because it makes all pods to be scheduled on one node.
Also it does not make any sense  when we use mysql for persistence
http://docs.grafana.org/installation/configuration/#database
http://docs.grafana.org/installation/configuration/#data

Reply:
Ok, makes sense. Thanks for the explanation.